services/analytics/base.go

```diff
@@ -144,3 +159,19 @@ func isDimension(name string) bool {
 	_, ok := dimensionColumnNames[name]
 	return ok
 }
+
+// isTimeMetric checks if a metric in the time metrics group.
+func isTimeMetric(name string) bool {
```

Comment:
maybe we can check for suffix? `wait` or `time`

Reply:
Yeah, I also thought about such a "hack" but decided that the current is more obvious.